src/exporters/insomnia/insomnia.ts

```diff
@@ -27,12 +27,12 @@ class Insomnia {
                         returnFields = this.buildReturnFields(returnType);
                     }
 
-                    if (query.type.ofType != null) {
+                    if (query.type.ofType != null && query.type.ofType.kind !== "LIST") {
```

Comment:
From what I could infer from [the docs](https://graphql.org/learn/introspection/), the `LIST` kind does not have a name, so entering this if branch was causing the script to crash in such cases